plugins/router/arris-tg3442

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python3
+
+"""
+# MUNIN Plugin to monitor status of Arris TG3442 / TG2492LG-85
```

Comment:
Since you are using a docstring (`"""`), you can remove the leading hashs (`#`).

Reply:
aye